client/src/pages/overview/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { Box, FormControl, InputLabel, MenuItem, Select } from "@mui/material";
+import Header from "components/Header";
+import OverviewChart from "components/OverviewChart";
+import Wrapper from "hoc/Wrapper";
```

Comment:
and btw, this should be  fixed as well as the rest of the components that don't have a persistent layout. Which is all of em